TI-reports/2024/2024-Q4-BEST-WG.md

```diff
@@ -0,0 +1,224 @@
+
+#### Current Status
+
+- TODO
+
+- #### Up Next
+
+- TODO
+
+
+### Developing Secure Software Fundamentals Course (LFD121)
+
+#### Purpose
+
+- Provide baseline security education for developers.
+
+#### Current Status
+
+- TODO
+
+#### Up Next
+
+- TODO
+
+
+### Concise Guides
+
+#### Purpose
+
+- Artifacts that consolidate BEST practices in OSS software development and management techniques
+
+#### Current Status
+
+- TODO
+
+#### Up Next
+
+- TODO
+
+
+### Compiler Hardening Guides
+
+#### Purpose
+
+- Help C and C++ developers and those who compile C/C++ code, e.g., package maintainers, ensure that produced application binaries (libraries and executables) are equipped with security mechanisms provided by compilers against potential attacks and/or misbehavior.
+
+#### Current Status
+
+- Continued revision, updates, & enhancement, e.g., keeping the compiler options hardening guide up-to-date with upstream options additions and changes in GCC and Clang/LLVM and addressing feedback from Linux distribution communities.
+
+#### Up next
+
+- Microsoft MSVC guidance planned for 2025 (tracked in [BEST Issue 150](https://github.com/ossf/wg-best-practices-os-developers/issues/150))
+- Plan outreach activities for 2025, possibly talks aimed at C++ conferences.  
+
+
+### EDU.SIG
+
+#### Purpose
+
+- Deliver Baseline Secure Software Development Education and Certification to All.  Provide access to open and widely available education materials to all learners.
+Materials will be maximally accessible and easy to consume for all learners.
+
+#### Current Status
+
+- TODO
+
+#### Up Next
+
+- TODO
+
+
+### Memory Safety SIG
+
+#### Purpose
+
+- The Memory Safety SIG is a group working within the OpenSSF's Best Practices Working Group formed to advance and deliver upon The OpenSSF's Mobilization Plan - Stream 4.
+
+#### Current Status
+
+- TODO
+
+#### Up Next
+
+- TODO
+
+
+### Python Hardening Guide
+
+#### Purpose
+
+- Help Python developers to create more secure code by explaining vulnerable and non-vulnerable coding patterns based on the CWE framework and rules.
+- Besides a description of each coding pattern, the guide includes executable code examples for each rule, which allow for an in-depth understanding of each pattern.
+
+#### Current Status
+
+- TODO
+
+#### Up Next
+
+- TODO
+
+
+### Scorecard
+
+#### Purpose
+
+- To help open source maintainers improve their security best practices and to help open source consumers judge whether their dependencies are safe.
+- Scorecard is an automated tool that assesses a number of important heuristics ("checks") associated with software security and assigns each check a score of 0-10.
+
+#### Current Status
+
+- TODO
+
+#### Up Next
+
+- TODO
+
+
+### Security Baseline
```

Comment:
Hello @eddie-knight @SecurityCRob! Can you please update the sections below with a status update? Thanks!